boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -611,7 +626,7 @@ PcMemGetMemoryMap(ULONG *MemoryMapSize)
         }
     }
 
-    TRACE("DiskReadBuffer=%p, DiskReadBufferSize=%lx\n",
+    TRACE("DiskReadBuffer=%p, DiskReadBufferSize=0x%lx\n",
```

Comment:
Then, do you want an added `0x` to `%p` here? (`0x%p` is used in the tree, but not much compared to `%p`...)

Reply:
I personally always put "0x" when I know I'm going to output hex.
Because if you see eg.
"DiskReadBufferSize = 3123", do you know at first sight whether this is in decimal or in hex?
When using the prefix you don't need to know what the code is doing to know it's hex.

Reply:
Added.